addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -175,14 +176,32 @@ def _compute_description_validity(self):
                 name_validity = _(
```

Comment:
we should use `self.env._` now for translation 
```suggestion
                name_validity = self.env._(
```